src/Sleep.js

```diff
@@ -1,32 +1,47 @@
 import sleepData from './data/sleep';
+// import averager from './util';
```

Comment:
These kind of comments I feel like are safe to remove since they are no longer being used by us, since they are no lingered referenced